lib/conduit_mqtt/conn.ex

```diff
@@ -28,7 +28,7 @@ defmodule ConduitMQTT.Conn do
 
   ## Server Callbacks
   def init(opts) do
-    # Process.flag(:trap_exit, true) #TODO what does this do?
+    #Process.flag(:trap_exit, true) #TODO what does this do?
```

Comment:
If a linked process crashes, it sends a message to this process to allow it to handle the crash rather than immediately crashing itself. Since you're not linking processes, you don't need it.

Reply:
Yes, it is. Since you're starting it that way, you probably want this process to crash so the supervisor will restart it